src/lib/modules.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+    Scrollbar: require('finbars')
```

Comment:
Is the idea here to eventually move all imports to this file? Just wondering why this abstraction